build/packager.js

```diff
@@ -257,12 +263,46 @@ Packager.prototype.package = function (next) {
 		this.generateManifestJSON.bind(this),
 		function (cb) {
 			console.log('Writing JSCA');
-			fs.copy(path.join(this.outputDir, 'api.jsca'), path.join(this.zipSDKDir, 'api.jsca'), cb);
+			gulp.src(path.join(this.outputDir, 'api.jsca'))
+				.pipe(changed(this.zipSDKDir))
+				.pipe(gulp.dest(this.zipSDKDir))
+				.on('end', cb);
 		}.bind(this),
 		function (cb) {
 			console.log('Copying SDK files');
-			// Copy some root files, cli/, common/, templates/, node_modules minus .bin sub-dir
-			this.copy([ 'CREDITS', 'README.md', 'package.json', 'cli', 'common', 'node_modules', 'templates' ], cb);
+			// Copy some root files, cli/, templates/, node_modules
+			gulp.src([ 'CREDITS', 'README.md', 'package.json', 'cli/**', 'node_modules/**', 'templates/**' ], {
+				cwd: this.srcDir,
+				base: '.'
+			})
+				.pipe(changed(this.zipSDKDir))
+				.pipe(gulp.dest(this.zipSDKDir))
+				.on('end', cb);
+		}.bind(this),
+		function (cb) {
+			console.log('Transpiling common SDK JS');
+			const destDir = path.join(this.zipSDKDir, 'common');
+			// Pull out android's V8 target (and transform into equivalent chrome version)
+			const v8Version = require('../android/package.json').v8.version;
+			const found = v8Version.match(V8_STRING_VERSION_REGEXP);
+			const chromeVersion = parseInt(found[1] + found[2]); // concat the first two numbers as string, then turn to int
+			// Now pull out min IOS target
+			const minSupportedIosSdk = version.parseMin(require('../iphone/package.json').vendorDependencies['ios sdk']);
+			gulp.src(this.srcDir + '/common/**/*.js')
+				.pipe(changed(destDir))
+				.pipe(gulpBabel({
+					presets: [
+						[ '@babel/env', {
+							targets: {
+								// TODO: filter to only targets relevant for platforms we're building?
+								ios: minSupportedIosSdk,
+								chrome: chromeVersion
```

Comment:
Do we maybe want to add safariVersion for Windows too? Just incase iOS moves past that? It should be `safariVersion: '10'` based off the [current code](https://github.com/appcelerator/titanium_mobile_windows/blob/2ae2ef9c6e601181356a0ffc9e43ab1b6b851a71/cli/commands/_build/copy.js#L629)

Reply:
my concern here is that e don't extract that version out to the windows package.json right now, it appears to be hard-coded. We should extract it out and then we can reference it from the same place.